package.json

```diff
@@ -13,9 +13,10 @@
   ],
   "scripts": {
     "test": "jest",
-    "build": "tsc",
-    "prepublish": "tsc",
-    "lint": "eslint src --ext .ts"
+    "build": "tsc -p tsconfig.build.json",
+    "prepublish": "rm -rf dist && yarn lint && yarn build && yarn test",
+    "lint": "eslint --ext .ts src",
+    "lint:fix": "eslint --fix --ext .ts src"
```

Comment:
Should we add `format` here?

Reply:
I added the eslint plugin for prettier. Therefore eslint checks prettier and fix reformats the code.